packages/app/init/init.js

```diff
@@ -4,18 +4,20 @@
  */
 
 /**
- * @typedef InitializationObject
- * @property {object} config - The key/value configuration inside the initialization object
- * @property {string} config.api_key - A string of key that will be used to access inLive protected API
- * @property {string} name - The name of initialization object instance
+ * A function to create object
+ *
+ * @param {Config} config -- being passed from init module parameter
  */
+function initializeConfig(config) {
```

Comment:
I think `InitializationObject` is more appropriate because this constructor function will be referenced in other modules for instance checking purposes. Or do you have other suggestions? What about `InliveInstance` or `InitializationInstance`?

The name should use PascalCase because this is categorized as constructor function which is the same as creating a class object.

Reply:
@faiq-naufal Yeah, I kinda confused what kind of name for this function actually. If compared to mas faiq options, I think I prefer to use `InitializationInstance` because as per mas faiq info that this is for instance checking & I think `Initialization` is a good way to refer to this initial thing module.

On process to rename it.

Reply:
@faiq-naufal done at commit [f3459a2](https://github.com/inlivedev/inlive-js-sdk/pull/14/commits/f3459a280920e48847fd4effb60a07fbe81fa770)

Reply:
Ah sorry one more thing, if the name changed to InitializationInstance, we need to update any words that have InitializationObject or initialization object to be initialization instance

Reply:
@faiq-naufal is it okay though for mas faiq? ^^"
I've revised it to be InitializationInstance on commit [509d7f1](https://github.com/inlivedev/inlive-js-sdk/pull/14/commits/509d7f1dddee139758d487f4d487c1cbd577c1ec), but if too much on your side, that's okay if revise it again to be `InitializationObject`..

Reply:
It's fine. Only a few on my sides